tutorpicasso/commands/enable_private_packages.py

```diff
@@ -35,14 +37,12 @@ def enable_private_packages() -> None:
                 f"{package} is missing one of the required keys: 'name', 'repo', 'version'"
             )
 
-        if os.path.isdir(f'{private_requirements_path}/{info["name"]}'):
-            subprocess.call(
-                ["rm", "-rf", f'{private_requirements_path}/{info["name"]}']
-            )
+        requirement_path = f'{private_requirements_path}/{info["name"]}'
+        if os.path.isdir(requirement_path):
+            tutor_utils.execute("rm", "-rf", requirement_path)
 
-        subprocess.call(
-            ["git", "clone", "-b", info["version"], info["repo"]],
-            cwd=private_requirements_path,
+        tutor_utils.execute(
+            "git", "clone", "-b", info["version"], info["repo"], requirement_path
```

Comment:
Why do we use `requirement_path` and not `private_requirements_path` as before?

Reply:
Because I changed the execution function.
Before, with `subprocess.call` I used the `private_requirements_path` as a working directory.
But, in `tutor_utils.execute` I am using the git clone command as `git clone -b <branch> <repo> <directory>`, so in the directory, I need to use a directory in which I will clone, in this case: private_requirements_path/name_of_the_package. It is in line 40.